src/OmniSharp/Models/v1/DnxProject.cs

```diff
@@ -12,6 +13,7 @@ public class DnxProject
         public IList<string> Configurations { get; set; }
         public IList<string> ProjectSearchPaths { get; set; }
         public IList<string> Frameworks { get; set; }
+        public IList<DnxFramework> DnxFrameworks { get; set; }
```

Comment:
Not so sure about having both Frameworks and DnxFrameworks here. I think I'd be happier with just

```cs
public IList<DnxFramework> Frameworks { get; set; }
```

and ditching the old IList<string> - but that would mean breaking backwards compatibility /cc @jrieken